auction-server/src/state.rs

```diff
@@ -111,6 +111,10 @@ pub struct SimulatedBid {
     /// The profile id for the bid owner.
     #[schema(example = "", value_type = String)]
     pub profile_id:      Option<models::ProfileId>,
+    /// The gas limit for the contract call.
+    #[schema(example = "", value_type = String)]
```

Comment:
I don't think we are exposing the SimulatedBid anywhere in our APIs, so we don't need to provide examples or if you want to do it, use a valid example.